setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    name="ibm-fms-extras",
```

Comment:
probably don't need the `ibm` prefix if no name conflicts on pypi

Reply:
fixed in latest commit